src/modules/project/pages/projectView/hooks/useProjectGoals.tsx

```diff
@@ -48,6 +49,17 @@ export const useProjectGoals = (projectId: string | number | undefined) => {
     },
   })
 
+  const [updateProjectGoalOrdering] = useMutation(MUTATION_UPDATE_PROJECT_GOAL_ORDERING)
+
+  const handleUpdateProjectGoalOrdering = async (projectGoalIdsOrder: number[], projectId: string) => {
+    await updateProjectGoalOrdering({
+      variables: { input: { projectId, projectGoalIdsOrder } },
+      onCompleted() {
+        refetch()
+      },
+    })
+  }
+
```

Comment:
I see you did use the generated hooks, why? It keeps the responses typed, that makes development easier. Also,

What you've done is good too, but I like when the completed scenario doesnot have to be associated in where the mutation is called. We want it to be fetched after the completion of the mutation, we define all those parameters where we define the mutation. Keeps things cleaner. With this, you don't need async await in where you call the mutation.

```suggestion
  const [updateProjectGoalOrdering] = useUpdateProjectGoalOrderingMutation({
  onCompleted(){
     refetch()
  }
  })

  const handleUpdateProjectGoalOrdering = (projectGoalIdsOrder: number[], projectId: string) => {
   updateProjectGoalOrdering({
      variables: { input: { projectId, projectGoalIdsOrder } }
    })
  }

```